projects/novo-elements/src/services/novo-label-service.ts

```diff
@@ -21,6 +21,7 @@ export class NovoLabelService {
   dateAdded = 'Date Added';
   emptyTableMessage = 'No Records to display...';
   noMatchingRecordsMessage = 'No Matching Records';
+  noMoreRecordsMessage = 'No more records. Click "Previous" to go back.';
```

Comment:
When you don't know the total, you don't know if the next page has any records or not ahead of time, so we don't know if the next button should be disabled. We only know we've reached the end when you navigate to a not-first-page that has fewer records than the page size. I have no yet implemented a check for this, but could do it. Even if I add that check, there is always a chance that the last page could have exactly the number of records as the page size, in which case the next button will be active, and when the user clicks to the next page, it will be empty.